### PR TITLE
Cst 147 non school np qs

### PR DESCRIPTION
**Pull request by BroiSatse**
https://dfedigital.atlassian.net/jira/software/projects/CST/boards/119?selectedIssue=CST-147

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -29,6 +29,10 @@ en:
           attributes:
             chosen_provider:
               blank: Select whether or not you have chosen an NPQ and provider
+        forms/work_in_school:
+          attributes:
+            works_in_school:
+              blank: Select whether or not you currently work in a school, collego or academy trust?
```
Comment (fofr) [inline config/locales/en.yml:35]:
```suggestion
              blank: Select whether or not you currently work in a school, college or academy trust?
```

Typo

File: config/locales/en.yml [hunk 2]
```diff
@@ -43,6 +47,12 @@ en:
             email:
               blank: Email can’t be blank
               invalid: Enter a valid email address
+        forms/your_work:
+          attributes:
+            employer_name:
+              blank: Enter your employer's name
```
Comment (fofr) [inline config/locales/en.yml:53]:
```suggestion
              blank: Enter the name of your employer
```

File: config/database.yml [hunk 3]
```diff
@@ -20,6 +20,7 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  url: <%= ENV.fetch("DATABASE_URL", "postgres://localhost:5432") %>
```
Comment (tonyheadford) [inline config/database.yml:23]: Why was this needed?

**Review by tonyheadford — COMMENTED** [review]
Looks fine from a dev perspective, just one query

File: config/database.yml (hunk 3 above)
Comment (BroiSatse) [reply config/database.yml:23]: Previous setup assumed that everyone is running postgres as a system-level service. I'm running postgres via docker to handle multiple PG versions for different projects so I was unable to do anything without making this change

Comment (tonyheadford) [reply config/database.yml:23]: I thought just setting the `DATABASE_URL` in your local env would be enough and it would get picked up automatically?

**Review by tonyheadford — APPROVED** [review]
